From: Phillip Lougher <phillip@squashfs.org.uk>
Date: Thu, 6 Aug 2026 19:19:15 +0100
Subject: [PATCH] MAINTAINERS: remove git URL for Squashfs

The squashfs-next.git URL hasn't been updated for many years, and it now
doesn't exist.

So remove it from the MAINTAINERS entry.

Link: https://lore.kernel.org/20260806181916.617881-1-phillip@squashfs.org.uk
Signed-off-by: Phillip Lougher <phillip@squashfs.org.uk>
Cc: Derek Barbosa <debarbos@redhat.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 MAINTAINERS | 1 -
 1 file changed, 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index d28e9f59e9211..7afd6b353eaf2 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -25553,7 +25553,6 @@ M:	Phillip Lougher <phillip@squashfs.org.uk>
 L:	squashfs-devel@lists.sourceforge.net (subscribers-only)
 S:	Maintained
 W:	http://squashfs.org.uk
-T:	git git://git.kernel.org/pub/scm/linux/kernel/git/pkl/squashfs-next.git
 F:	Documentation/filesystems/squashfs.rst
 F:	fs/squashfs/
 
-- 
2.47.3

